packages/corvid-local-site/src/utils/wixFiles.js

```diff
@@ -1,12 +1,38 @@
 const { prettyStringify } = require("./prettify");
+const isString_ = require("lodash/isString");
+const flow_ = require("lodash/flow");
+const atob = require("atob");
+const btoa = require("btoa");
 
-const toWixFileContent = (content, documentSchemaVersion) =>
-  prettyStringify({
+const showRealJson = process.env.XPERT;
+
+const decodeContent = payload => {
+  const decode = flow_([atob, escape, decodeURIComponent, JSON.parse]);
+  return isString_(payload)
+    ? decode(payload)
+    : { ...payload, content: decode(payload.content) };
+};
+const encodeContent = content => {
+  const encode = flow_([JSON.stringify, encodeURIComponent, unescape, btoa]);
+  return !content.content
```

Comment:
it's here to differentiate between json and string files? ain't we wrap everything with content?

Reply:
Unfortunately, currently the obfuscated content is sometimes located in `content` but in other times (page files) it is located in `content.content`.
I think we should change that